lib/coinbase/address.rb

```diff
@@ -62,6 +62,27 @@ def balance(asset_id)
       Coinbase::Balance.from_model_and_asset_id(response, asset_id).amount
     end
 
+    # Enumerates the historical balances for a given asset belonging of address.
+    # The result is an enumerator that lazily fetches from the server, and can be iterated over,
+    # converted to an array, etc...
+    # @return [Enumerable<Coinbase::HistoricalBalance>] Enumerator that returns historical_balance
+    def hiatorical_balances(asset_id)
```

Comment:
Can we add a spec for this to our `shared_examples/address_balances`?

Reply:
added into the balance session, thanks a lot!